fashion-messaging-app-ui/src/App.jsx

```diff
@@ -24,7 +23,9 @@ export default function App() {
 
   useEffect(() => {
     setLoading(true);
-    fetch(`http://localhost:3000/photos?search=${searchQuery}`)
+    fetch(`http://localhost:3000/photos?search=${searchQuery}`, {
+      credentials: 'include', 
```

Comment:
I added this to store cookies across different requests in the browser. This helped get rid of the bug where the user was logged out upon refresh

Reply:
Nice!